src/test/java/seedu/address/model/AddressBookTest.java

```diff
@@ -83,12 +85,42 @@ public void getPersonList_modifyList_throwsUnsupportedOperationException() {
         assertThrows(UnsupportedOperationException.class, () -> addressBook.getPersonList().remove(0));
     }
 
+    @Test
+    public void randomOperations_ensureUniqueIDs() {
```

Comment:
`IDs` -> `Ids` to match our [coding style](https://se-education.org/guides/conventions/java/intermediate.html#:~:text=Abbreviations%20and%20acronyms%20should%20not%20be%20uppercase%20when%20used%20as%20a%20part%20of%20a%20name.)